setup.py

```diff
@@ -27,19 +27,12 @@
 from distutils.sysconfig import get_config_vars
 from Cython.Build import cythonize
 from Cython.Distutils import build_ext
+import glob
 
 import numpy as np
 
 npyver = int(np.__version__.split('.')[1])
```

Comment:
We don't use it anywhere else. Does it must be removed too?

Reply:
removed